go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cloudflare/terraform-provider-cloudflare
 go 1.15
 
 require (
-	github.com/cloudflare/cloudflare-go v0.17.0
+	github.com/cloudflare/cloudflare-go v0.17.1-0.20210602222010-1740533cf449
```

Comment:
are you able to remove this from the PR? we generally bump dependencies outside of the functionality and then just merge the feature once it is ready. this does mean we're pending the release of `cloudflare-go` but that is generally 👌

Reply:
I don't think I can that hash is the first commit containing the Latitude and Longitude fields on cloudflare.LoadBalancerPool they don't exist in the last released version.

Reply:
yep, that's expected. we'll have an error on the build here in GitHub but once we cut a new release of cloudflare-go, this will get the update and won't have to deal with conflicts as it will change underneath it. if we leave this here and in other PRs, all of them will have to resolve `go.mod` and `go.sum` conflicts upon update which are no fun for anyone.

when i pull the tests locally, i can shim in the latest master version too using `go`'s `replace` in the `go.mod` file to roughly check it out.

Reply:
makes sense I've removed the mod changes from this PR